packages/sdk/src/ZClientUtils.ts

```diff
@@ -41,7 +41,7 @@ export class ZClientUtils {
         let fileContent: string;
         try {
             fileContent = readFileSync(filePath, "utf-8");
-        } catch (error) {
+        } catch (_error) {
```

Comment:
Same as above - the error variable could be omitted
```suggestion
        } catch {
```